### PR TITLE
add: Only logging SQL statements

### LINKED ISSUE #1331: Simple way to see SQL statements
In EF6 it was very simple to output the generated SQL with a one-line of code using Database.Logging property.
I tried to do everything discussed in this article as well as ASP.NET Core logging linked article, but it doesn't work.
Could you please provide a COMPLETE sample of how to see the EF generated SQL in a ASP.NET Core Web API?
Thanks.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ed83069f-39a8-e992-95f7-a8b41154b6f4
* Version Independent ID: 144fe168-080f-6557-e4b2-a18adb4b89ed
* Content: [Logging - EF Core](https://docs.microsoft.com/en-us/ef/core/miscellaneous/logging)
* Content Source: [entity-framework/core/miscellaneous/logging.md](https://github.com/aspnet/EntityFramework.Docs/blob/master/entity-framework/core/miscellaneous/logging.md)
* Product: **entity-framework**
* GitHub Login: @rowanmiller
* Microsoft Alias: **divega**

### PR DESCRIPTION
**Pull request by SimonCropp**
fixes #1331. or at least show how complex it is to achieve

### CODE REVIEW

**Comment by ajcvickers** [comment]
@divega It shouldn't be this hard; we also need to make sure we're using the new 3.0 patterns for logging effectively. This probably requires a bit more investigation.

**Comment by SimonCropp** [comment]
As it stands, if this is currently the best way, shouldn’t this be merged?

File: samples/core/Miscellaneous/Logging/Logging/SqlCommandLoggingProvider.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            {
+                return this;
+            }
+
+            return NullLogger.Instance;
+        }
+
+        public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
+        {
+            if (eventId.Id != RelationalStrings.LogRelationalLoggerExecutedCommand.EventId.Id)
```
Comment (SimonCropp) [inline samples/core/Miscellaneous/Logging/Logging/SqlCommandLoggingProvider.cs]: So i note this is now broken in the latets EF beta. What is the replacement for `RelationalStrings.LogRelationalLoggerExecutedCommand`?

Comment (SimonCropp) [reply samples/core/Miscellaneous/Logging/Logging/SqlCommandLoggingProvider.cs]: seems it is now `RelationalEventId.CommandExecuting`

**Pull request closed without merging**